.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -0,0 +1,70 @@
+name: Bug Report
+description: Something isn't working as expected
+labels: [bug]
+body:
+- type: markdown
+  attributes:
+    value: "## Thank you for contributing to Swift-DocC!"
+- type: markdown
+  attributes:
+    value: |
+      Before you submit your issue, please complete each textarea
+      below with the relevant details for your bug, and complete
+      the steps in the checklist
+- type: textarea
+  attributes:
+    label: Description
+    description: |
+      A short description of the incorrect behavior. 
+      If you think this issue has been recently introduced and did not occur in an 
+      earlier version, please note that. If possible, include the last version that 
+      the behavior was correct in addition to your current version.
```

Comment:
It looks like GitHub doesn't handle line breaks in the way I would expect for markdown content.

Here's what I'm seeing in the testing repo you put up:

<img width="775" alt="Screen Shot 2022-07-01 at 10 07 41 AM" src="https://user-images.githubusercontent.com/6750147/176939644-c53e5dcc-8cdb-4e03-961a-a40626290e90.png">

Maybe we should condense into a single line here (and elsewhere) so that it renders more nicely?

```suggestion
      A short description of the incorrect behavior. 
      
      If you think this issue has been recently introduced and did not occur in an earlier version, please note that. If possible, include the last version that the behavior was correct in addition to your current version.
```